pylink/unlockers/__init__.py

```diff
@@ -31,6 +31,6 @@ def unlock(jlink, name):
     Raises:
       NotImplementedError: if no unlock method exists for the MCU.
     """
-    if name.lower() == 'kinetis' or name.lower() == 'freescale':
+    if name.lower() in ['kinetis', 'freescale', 'nxp']:
```

Comment:
@frojasg Out of curiosity, why did we add `freescale` here? Was this new in 6.x?

Reply:
@frojasg Yes.  For a brief period, some of the 6.x versions listed the Kinetis devices as Freescale.  This was later fixed.